### ISSUE TITLE
first() does not respect orderBy

### ISSUE THREAD

**Issue by silentlight** [revolver-app/vuex-orm#20]
Hi @kiaking 

I have a following query: 
```
const lastTimeLog = this.$store.getters['entities/timeLogs/query']()
          .orderBy('stopped_at', 'desc')
          .first();
```

first() returns always first entity without respecting order. From the code above I expect to get time log with biggest value for stopped_at. Instead, I always get first entity. Am I missing something?

**Comment by silentlight** [comment]
A workaround would be to do this:
```
const timeLogs = this.$store.getters['entities/timeLogs/query']()
          .orderBy('stopped_at', 'desc')
          .get();

return timeLogs.length ? timeLogs[0] : null;
```

But since we have `first()` I thought that I can use it in conjunction with `orderBy`.

**Comment by kiaking** [comment]
Wow this sounds like a bug. I'll look into it! (If it's a bug, I think it wouldn't take long to fix it).

**Comment by silentlight** [comment]
Great! Thanks @kiaking

**Comment by kiaking** [comment]
This issue has been fixed with VUEx ORM 0.11.4!

@silentlight Thank you for reporting this issue!
